client_youtube.py

```diff
@@ -161,7 +157,7 @@ def _parse_subs(filename) -> List[SubtitleRange]:
                 )[0]
             except IndexError:
                 break  # reached end of subs
-            line = "".join(chunk.split("\n")[1:])
+            line = " ".join(chunk.split("\n")[1:])
```

Comment:
This fixes the problem of the subtitles concatenated incorrectly like `wordAwordB` -> `word A word B`.